src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -161,6 +161,45 @@ export default async function UserBreaches() {
     return breachRowsHTML;
   }
 
+  // Fetch list of subscriptions.
+  const bearerToken = session?.user.subscriber?.fxa_access_token;
+  if (!bearerToken) {
+    console.error("User has no bearer token");
```

Comment:
I think this relates to a previous comment, but if there is no bearer token, will the `fetch()` work, or should we just add some code here to abort early?

Although, I guess if the `bearerToken` is falsy here, then the `else` block on L182 below happen (which also just logs to STDERR and should leave `subscriptions` as `undefined` which then gets passed into `userSubscribed()` and I guess causes the "Subscribe to Premium" link to get rendered.